src/zookeeper.jute

```diff
@@ -51,6 +51,10 @@ module org.apache.zookeeper.data {
         long ephemeralOwner; // owner id if ephemeral, 0 otw
         long pzxid;      // last modified children
     }
+    class PathWithStat {
+        ustring path;
+        Stat  stat;
```

Comment:
Please modify Stat stat; member like below for consistency.

org.apache.zookeeper.data.Stat stat;